README.md

```diff
@@ -92,10 +60,25 @@ show files inside of visual studio correctly (This will be added later)
 
 ## Contributing
 
-The source files in this repo are for historical reference and will be kept static, and this repository will
-be archived. Feel free to fork this repo and experiment.
+3DMMForever and 3DMMPlus will not be possible without an enthusiastic open source developer community backing it.
+
+We're working on a set of contribution guidelines that we will be using going forward.
 
-## Code cleanup
+## 3DMMPlus
+
+3DMMPlus is a future fork of 3DMMForever that will be created when we've finished the porting work to MacOS and Linux. The sky will be the limit with 3DMMPlus! Our mission will be to create a heavily enhanced version of 3DMM that:
+
+- Has a flexible full color UI that looks great at modern resolutions.
+- Has a modern full color 3D renderer with lightning, shading, moveable camera, and more.
```

Comment:
Frank can you go through my github issues and ideas and outline and discuss with the maintainers my personal pet peeves of using this software for 10+ years animating.  Other animators in the community have feedback as well for their hopes and dreams for 3DMMPlus.

Ideally we should maybe make some kind of shared community "+1" where coders like yourself can evaluate what the actual animators also want because we make more movies than you hackers do and have specific needs as well.  So is there a good way to prioritize issues with some app or excel spreadsheet somewhere we can start with this list and my issues list and try to get the "top most requested and wanted features" based on a voting system.

We could ensure that only people can vote once per issue or something like a reddit post comment or something right?

Let me know your thoughts on the community approach to the features we provide in 3DMMPlus because in my opinion if someone like myself animates something by hand with raw 3dmm with no knowledge of v3dmm or any tools for 3 years

https://www.youtube.com/watch?v=u1cHqXaSO7Y

I created perfect dark and its my masterpiece.  I probably animated for 2000 hours on it using 3dmm.  Its the creators like myself who should have the most influence on the tooling to create animations more efficient dont you agree?

OMG im getting so excited of the prospect of something with the simplicity of 3dmm's origin, but enhanced to new levels.

I can help code this voting system somehow to help, but I think its necessary for everyone to add new feature possibilities and for everyone to review all features/bug fixes and caveats to animating in this childrens program that we (well mostly you and foone and other maintainers who know crazy C stuff) are going to knock out over the years and I am going to test builds.

I cant wait to test every nook and cranny and review all your guys PRs to understand all components and how your code can impact things!

Please advise my friend on this idea of features/bugs for 3DMMPlus before this PR is merged.